cmd/cloudinitsave/cloudinitsave.go

```diff
@@ -46,73 +48,42 @@ const (
 	datasourceInterval    = 100 * time.Millisecond
 	datasourceMaxInterval = 30 * time.Second
 	datasourceTimeout     = 5 * time.Minute
+	configDev             = "LABEL=config-2"
+	configDevName         = "config-2"
+	configDevMountPoint   = "/media/config-2"
```

Comment:
so I'd make the derivative 2 look like `configDev= "LABEL="+configDevName` - no big stress tho